src/applications/disability-benefits/view-payments/README.md

```diff
@@ -0,0 +1,31 @@
+App Name: `View Payments`
+Active engineers: Micah Chiang (front end), Jesse Cohn (front end), Kathleen Crawford (back end), Kevin Musiorski (back end)
+Form ID (if different from app name, NA otherwise): `NA`
+URL: `{root url}/va-payment-history/payments`
+
+
+# Background
+Veterans receive lots of payments for the various benefits they receive from the VA. This app allows Veterans to see what payments have been made to them as well as some basic information about those payments.
+
+## How the app works for the user
+When the user lands on the View Payments page they will either have payments or not. If they have no payments they will be shown a message that we have no payments on file for them. If they do have payments they will be shown a list of all the payments we have on file for them. If the user does have payments they will be split into two lists, one is payments that the Veteran has received, the second is payments that have been returned to the VA.
+
+The page also includes some static content about payments with helpful links.
+
+> There is a third possibility when the user lands on the page, that we had an error looking up their payments. If there is an error they will be shown a message about the error.
+
+## The front end code
+The front end is split up into a few basic components that are inside a container that acts as a wrapper and connects the app to our Redux code. The container, located at [/containers/App.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/a4babda01dac9cbb30feded94e92ea8f557b69be/src/applications/disability-benefits/view-payments/containers/App.jsx#L1), simply takes the main components and wraps them in a `<DowntimeNotification />` from the platform in case BGS is down (which we use as the data source on the back end). This is then wrapped in a `<RequiredLoginView />` component from the platform that simply requires the user to log in to view their payments. You will also notice the `mapStateToProps` call where we tie the app into Redux.
+
+### Payment lists
+The meat inside the `App.js` container is the `<ViewPaymentsLists />` located at `/components/view-payments-lists/ViewPaymentsLists.jsx`. This component is responsible for rendering most of the content on the page. It begins with a call to [getAllPayments](https://github.com/department-of-veterans-affairs/vets-website/blob/a4babda01dac9cbb30feded94e92ea8f557b69be/src/applications/disability-benefits/view-payments/actions/index.js#L20) inside a `componentDidMount` which calls the action from Redux passed in through `mapDispatchToProps`. This action calls the `/profile/payment_history` endpoint and gets the payments for the Veteran.
```

Comment:
I think it'd be worthwhile to link `/profile/payment_history` to the [swagger docs](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/profile/getPaymentHistory) or the controller itself. Swagger docs are probably fine though.

Reply:
Derp, just noticed you've linked to the controller down below. Still could be worthwhile dropping the swagger docs link somewhere, since it contains the structure of the payload the frontend receives.